launchable/utils/http_client.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import os
+import platform
+from launchable.version import __version__
+
+
+BASE_URL_KEY = "LAUNCHABLE_BASE_URL"
+DEFAULT_BASE_URL = "https://api-static.mercury.launchableinc.com"
```

Comment:
@ninjinkun The default base URL should be `https://api.mercury.launchableinc.com`. `api-static` subdomain is used only when you want static IPs.

Reply:
Oh, I see